components/UiKit/LinkAsButton.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from "styled-components";
+
+const LinkAsButton = styled.a`
```

Comment:
This is fine for now, but we will probably change this to a generic button component.
If you want to use the styled.button as a `<a>` tag, you can just use it like this: 
`<Button as="a" href="#">Link with Button styles</Button>`

See documentation: https://styled-components.com/docs/api#as-polymorphic-prop